minidump/src/minidump.rs

```diff
@@ -3946,8 +3946,21 @@ fn read_string_list(
     let count: u32 = data
         .gread_with(&mut offset, endian)
         .or(Err(Error::StreamReadFailure))?;
+    let count = count as usize;
```

Comment:
since you bother doing a `.checked_mul()` it seems weird to not use `TryInto` here instead of the cast

Reply:
I am casting from `u32` to `usize`. I don’t know if Rust even supports any platform for which that is fallible.

Reply:
rust has community support for a few 16-bit microcontrollery things but realistically we don't need to worry about compat with that kind of platform.